README.md

```diff
@@ -48,6 +48,33 @@ The following changes are a **VIOLATION** and **ARE NOT** included:
   * ❌ Re-engaging of <ins>**LONGITUDINAL CONTROL**</ins> (acceleration/brake) on brake pedal release ❌ (was never included in MADS)
   * ❌ No disengaging on ACC MAIN in OFF state ❌
 
+⚒ Installation
+---
+
+### Comma URL (Comma 3 only quick setup)
+* Test-C3 is more stable than 8.14
```

Comment:
Test-C3 -> `test-c3` (encapsulate in ` ). Also I would actually remove this line or swap it around as `test-c3` is _less_ stable compared to 0.8.14 IMO.

Reply:
No problem! Sounds good and looks good. Thanks for the revisions